apps/demo-ui/ui/src/Chatbox.tsx

```diff
@@ -927,6 +928,20 @@ export const ChatBox = ({ chatHistory, searchResults, setChatHistory, setSearchR
                 setDocsLoading(false)
             }
             const startTime = new Date(Date.now());
+            const truncateString = (chatInput: string, maxLength = 15) => {
```

Comment:
why 15?

Reply:
15 char length seemed an appropriate fit in the conversation list item otherwise it was overflowing.